app/org/sagebionetworks/bridge/services/AuthenticationServiceImpl.java

```diff
@@ -210,8 +213,12 @@ public void requestResetPassword(Study study, Email email) throws BridgeServiceE
         checkNotNull(email);
         
         Validate.entityThrowingException(emailValidator, email);
-
-        accountDao.requestResetPassword(study, email);
+        try {
+            accountDao.requestResetPassword(study, email);    
+        } catch(EntityNotFoundException e) {
+            // Suppress this. Otherwise it reveals if the account does not exist
+            logger.info("Request reset password attempt using address: "+study.getIdentifier()+" "+email.getEmail());
```

Comment:
This would indicate whether a person was registered or not, if I understand correctly, as a sign up starts with the API, not with an email to that email address (due to needing a password as you say).